Rise from Exile (#438)
qwrtln/Homm3BG-mission-book: draft-scenarios/clash/rise_from_exile.tex

% !TeX spellcheck = en_US

> \addscenariosection{1}{Clash Scenario}{Rise from Exile}{\images/logistics.png}

Reviewer: Logistics is already used, maybe another image?

Author: yeah, that was just a place holder for now. I have to figure out another image..

\begin{multicols}{2}

\textbf{Author:} Laamakala

\textit{Cast out and crownless, Heroes wander the ruined frontier, seeking strength, allies, and the road back to their stolen throne.}

\subsection*{\MakeUppercase{Scenario Length}}
This Scenario plays out over 10 Rounds.

\subsection*{\MakeUppercase{Player Setup}}

> \textbf{Player Count:} 2-4

Reviewer: ```suggestion
\textbf{Player Count:} 2 -- 4
```

\textbf{Starting Resources:} 18 \svg{gold}

\textbf{Starting Income:} 10 \svg{gold}, 0 \svg{building_materials}, 0 \svg{valuables}

\textbf{Starting Units:}
\begin{itemize}

>   \item Search (3) Neutral \svg{bronze} Deck and place 2 chosen different Stack Tokens

Reviewer: ```suggestion
  \item Search (3) Neutral \svg{bronze} Units and place 2 chosen different Stack Tokens on them
```
Is there enough space to make it explicit?

  \item Search (2) Neutral \svg{silver} Units
\end{itemize}

\textbf{Town Buildings:}
\begin{itemize}
  \item Mage Guild \textit{or} \svg{building_special_tent} Building
  \item City Hall
\end{itemize}

\subsection*{\MakeUppercase{Map Setup}}
Take the following Map Tiles and arrange them as shown in the Scenario map layout ($P$ stands for the number of players):

\begin{itemize}
  \item $P$ × Starting (I) Map Tiles
  \item 3$P$ × Far (II--III) Map Tiles
  \item $P$ × Near (IV--V) Map Tiles (with Obelisk)
  \item 6 × Near (IV--V) Map Tiles (without Obelisk)
  \item 1 × Center (VI--VII) Map Tile
\end{itemize}

\subsection*{\MakeUppercase{Victory Condition}}
Score the most Victory Points (VPs) by the end of the \nth{10} Round. In case of a tie, draw Cards to your \svg{hand} Limit and engage in a final Combat -- the winner is the victor.

\subsection*{\MakeUppercase{Victory Points}}

> The player with the most Victory Points (VPs) wins.

Reviewer: ```suggestion
```
This is already stated in the victory condition above.

\begin{itemize}
  \item 4 VPs for the first player to flag the Center VII Field.
  \item 2 VPs for each enemy Main Hero defeated.
  \item 1 VP for each flagged Obelisk.
\end{itemize}

\subsection*{\MakeUppercase{Timed Events}}
\textbf{\nth{1} and \nth{2} Round:}
\begin{itemize}

>   \item All Secondary Heroes gain +1 \svg{movement}.

Reviewer: ```suggestion
  \item All Secondary Heroes gain +1~\svg{movement}.
```
Use a hard space in such cases.

\end{itemize}
\textbf{\nth{3} Round:}
\begin{itemize}
  \item Build either \svg{building_special_tent} or \svg{building_special_gears} Building in all uncaptured Starting Towns for free, ignoring any

> prerequisite buildings, and add a ``Few'' \svg{bronze} Units to the guards. Prerequisite buildings are not granted and must be built

Reviewer: "guards" feel out of place here. If they are explained in additional rules below, it should be mentioned here (something like "explained below"). See how Wandering Dragons do this regarding dragon movement.

  \item Until their Starting Town has been captured, Secondary Heroes may gain rewards from each \textbf{Visitable} Field \textbf{twice}.
  \item Once per Round, Secondary Heroes may discover a Map Tile for 0 \svg{movement}.
  \item Your M\&M Deck acts as your Secondary Hero's Hand.  Do not draw Cards at the beginning of your turn. After capturing your Starting Town, use the M\&M Deck normally with your Main Hero.
  \item After winning a Neutral Combat with your Secondary Hero, you may recruit one of the defeated Neutral Units by paying its recruitment cost.
  \item Players may not use/build Buildings or recruit Units from their Starting Town until they capture it.
  \item Extending Starting Town Combat costs 0 \svg{movement}.

\columnbreak

  \item \textbf{Starting Towns} are guarded by:
  \begin{itemize}
    \item 2 × A ``Pack'' of the cheapest \svg{bronze} Units
    \item A ``Few'' of the most expensive \svg{silver} Units
  \end{itemize}
  \item If a player loses Combat against their Starting Town, any defeated guards do not return.
  \item After capturing the Starting Town:
  \begin{itemize}

>     \item Gain your Main Hero. Gain Levels equal to the Round number \textit{(max 4)} and draw Cards to your \svg{hand} Limit.

Reviewer: What does it mean exactly? If you're Level 3 on the 4th round, you advance to level 4 or level 7?

Author: You start with secondary hero -> no gaining levels. after you capture the starting town, you gain your main hero (= level 1) -> level up equal to the round number.
So this needs rephrasing to either gain XP equal to the round number+1, or "Your Main Hero starts at the level equal to the Round number". not sure what is the best phrasing.
round 2 capture -> Main hero starts at level 2
round 3 capture -> Main hero starts at level 3
round 4+ capture -> Main hero starts at level 4

\end{itemize}

\columnbreak

\textbf{Optional Rule: Unit Specialty Hero}
\begin{itemize}
  \item If a player's starting Hero has a Unit Specialty, that player may choose their starting \svg{bronze} Unit to match that Specialty.
  \item If that Unit is the most expensive \svg{bronze} Unit of its Faction, that player's starting \svg{silver} Unit must cost 10 \svg{gold} or less.
  \item After choosing your Main Hero, you may add your starting Hero's Starting Ability Card, Specialty Card, or both to your Main Hero's Deck.
\end{itemize}

\textbf{Optional Rule: Main Hero Start}
\begin{itemize}
  \item Players start with their Main Hero instead of their Secondary Hero.
  \item Until a player captures their Starting Town, their Deck acts as a depleting Hand. Discarded Cards are not shuffled back into the Deck.
  \item When recruiting a Secondary Hero, choose one Hero from that Faction. You may add that Hero's Starting Ability Card, Specialty Card, or both to your Deck.
  \item When Leveling Up, you may choose which of your Heroes' next Specialty Card to add to your Deck.
\end{itemize}

\end{multicols}

> \includegraphics[width=\linewidth]{\art/harpy.jpg}

Reviewer: ```suggestion
\vspace*{\fill}
\includegraphics[width=\linewidth]{\art/harpy.jpg}
```
For some symmetry.

\subsection*{\MakeUppercase{Scenario Length}}
This Scenario plays out over 10 Rounds.

\subsection*{\MakeUppercase{Player Setup}}
\textbf{Player Count:} 2-4

\textbf{Starting Resources:} 18 \svg{gold}

\textbf{Starting Income:} 10 \svg{gold}, 0 \svg{building_materials}, 0 \svg{valuables}

\textbf{Starting Units:}
\begin{itemize}
  \item Search (3) Neutral \svg{bronze} Deck and place 2 chosen different Stack Tokens
  \item Search (2) Neutral \svg{silver} Units
\end{itemize}

\textbf{Town Buildings:}
\begin{itemize}
  \item Mage Guild \textit{or} \svg{building_special_tent} Building
  \item City Hall
\end{itemize}

\subsection*{\MakeUppercase{Map Setup}}
Take the following Map Tiles and arrange them as shown in the Scenario map layout ($P$ stands for the number of players):

\begin{itemize}
  \item $P$ × Starting (I) Map Tiles
  \item 3$P$ × Far (II--III) Map Tiles
  \item $P$ × Near (IV--V) Map Tiles (with Obelisk)
  \item 6 × Near (IV--V) Map Tiles (without Obelisk)
  \item 1 × Center (VI--VII) Map Tile
\end{itemize}

\subsection*{\MakeUppercase{Victory Condition}}

> \begin{itemize}

Author: ```suggestion
The game ends at the end of the Round when any of the following conditions are met:
\begin{itemize}
```

>   \item At the end of Round 10.

Author: ```suggestion
  \item \textbf{4 players:} One player has defeated every opponent's Main Hero at least once. \textit{That player wins the game immediately.}
```

>   \item \textbf{4 players:} One player has defeated every opponent's Main Hero at least once. \textit{That player wins the game immediately.}

Author: ```suggestion
    \item At the end of Round 10.
```

\begin{itemize}
  \item 4 VPs for the first player to flag the Center VII Field.
  \item 2 VPs for each enemy Main Hero defeated.
  \item 1 VP for each flagged Obelisk.
\end{itemize}

\subsection*{\MakeUppercase{Timed Events}}
\textbf{\nth{1} and \nth{2} Round:}
\begin{itemize}
  \item All Secondary Heroes gain +1 \svg{movement}.
\end{itemize}
\textbf{\nth{3} Round:}
\begin{itemize}
  \item Build either \svg{building_special_tent} or \svg{building_special_gears} Building in all uncaptured Starting Towns for free, ignoring any
prerequisite buildings, and add a ``few'' \svg{bronze} Units to the guards. Prerequisite buildings are not granted and must be built
separately to use their effects.
\end{itemize}
\textbf{\nth{4} Round:}
\begin{itemize}
  \item Build \svg{bronze} Dwelling in all uncaptured Starting Towns for free and Reinforce the guarding \svg{bronze} Units to ``Pack''.
\end{itemize}
\textbf{\nth{5} Round:}
\begin{itemize}
  \item Build Citadel in all uncaptured Starting Towns for free and Reinforce the guarding \svg{silver} Units to ``Pack''.
\end{itemize}

\columnbreak

\subsection*{\MakeUppercase{Additional Rules}}
\begin{itemize}

>   \item \textbf{Optional:} Use random Starting Tiles.

Author: ```suggestion
```

  \item All players start with only their Secondary Hero on an empty Field on a Far Map Tile marked \textbf{``S''} that contains a Settlement.

>   \item All Starting Tiles start undiscovered.

Author: ```suggestion
```

  \item Until their Starting Town has been captured, Secondary Heroes may gain rewards from each \textbf{Visitable} Field \textbf{twice}.
  \item Once per Round, Secondary Heroes may discover a Map Tile for 0 \svg{movement}.

>   \item Your M\&M Deck acts as your Secondary Hero's Hand. Discarded Cards remain in the Discard Pile until you capture your Starting Town. After that, use the M\&M Deck normally with your Main Hero.

Author: ```suggestion
  \item Your M\&M Deck acts as your Secondary Hero's Hand.  Do mot draw Cards  at the beginning of your turn. After capturing your Starting Town, use the M\&M Deck normally with your Main Hero.
```

  \item After winning a Neutral Combat with your Secondary Hero, you may recruit one of the defeated Neutral Units by paying its recruitment cost.
  \item Players may not use/build Buildings or recruit Units from their Starting Town until they capture it.
  \item Extending Starting Town Combat costs 0 \svg{movement}.

\columnbreak

  \item \textbf{Starting Towns} are guarded by:
  \begin{itemize}
    \item 2 × A ``Pack'' of the cheapest \svg{bronze} Units
    \item A ``few'' of the most expensive \svg{silver} Units
  \end{itemize}
  \item If a player loses Combat against their Starting Town, any defeated guards do not return.
  \item After capturing the Starting Town:
  \begin{itemize}
    \item Gain Main Hero. Gain Levels equal to the Round number \textit{(max 4)} and draw Cards to your \svg{hand} Limit.
    \item Gain 10 \svg{gold}, 2 \svg{building_materials}, and 1 \svg{valuables}.
    \item Gain 2 × A ``Pack'' of the cheapest \svg{bronze} Units.
    \item Place 1 Stack Token on one \svg{bronze} Unit.
  \end{itemize}

>   \item Defeated Main Heroes may spawn from a visited Obelisk with +1 \svg{movement}.

Author: ```suggestion
```
